app/controllers/user_challenges_controller.rb

```diff
@@ -27,6 +28,28 @@ def destroy
     end
   end
 
+  def evaluate_comments
+    evaluator = UserChallenge::Evaluator.new(params: params)
+    result = evaluator.save_comments
+
+    if result.success?
+      head :no_content
+    else
+      render json: { resource: 'user_challenge', errors: [user_challenge: result.errors.first] }, status: 500
```

Comment:
👋 Hi @atareshawty!

So I'm just going to swap out the status codes for my actions for this PR.

I saw a few other things ([needz moar transaction](https://github.com/osumb/challenges/blob/rails/app/controllers/user_challenges_controller.rb#L34)) while I was going to do that to the other 500s, and to avoid bloating this PR I'm just going to open a new PR and do some boy scouting.